specifications/crypto/spec.md

```diff
@@ -22,30 +22,30 @@ for the consensus and for signing transactions.
 #### Algorithmic choices
 
 The Ed25519 algorithm is standardized by the Internet Engineering Task Force (IETF) in
-[RFC-8032](https://tools.ietf.org/html/rfc8032) and in [FIPS
+[RFC-8032](https://tools.ietf.org/html/rfc8032) and is about to be, in [FIPS
 186-5](https://csrc.nist.gov/publications/detail/fips/186/5/draft). The
 corresponding OID is [1.3.101.112](http://oid-info.com/get/1.3.101.112).
 
 Libra's implementation of EdDSA makes choices and slight variations from these
 standards in ways that we make precise below:
 
-* Libra uses the PureEdDSA variant of Ed25519 described in RFC 8032. IndeedBoth
-  RFC 8032 and FIPS 186-5 also specify a pre-hashing mode for their variants,
-  whereby the message `m` taken as input to the signature algorithm is replaced
-  by a pre-hashed image `PH(m)`, with `PH` a customizable hash function. We do
-  not use this legacy interface.
+* Libra uses the PureEdDSA variant of Ed25519 described in RFC 8032. Indeed,
+  both RFC 8032 and FIPS 186-5 also specify a pre-hashing mode for their
+  variants, whereby the message `m` taken as input to the signature algorithm
+  is replaced by a pre-hashed image `PH(m)`, with `PH` a customizable hash
+  function. We do not use this legacy interface.
 * Both on signature verification and deserialization, we do perform a bounds
   check on the second half of each signature, as required in RFC 8032 §3.4 and
   FIPS186-5 §7.7-1. That is, if a signature is `R∥S`, with `R` and `S` 32-byte
   bytestrings, and `∥` the concatenation, we reject signatures for which the
   scalar `S` is not within the interval [0 ... L-1], with L the order of the
-  prime order subgroup of the field.<sup>[1](#f1)</sup> The original EdDSA
-  paper<sup>[2](#f2)</sup> and older Ed25519 libraries do not always perform
-  this check, which is necessary to avoid malleability.
+  prime order group generated by the curve's base point.<sup>[1](#f1)</sup> The
+  original EdDSA paper<sup>[2](#f2)</sup> and older Ed25519 libraries do not
+  always perform this check, which is necessary to avoid malleability.
 * When deserializing points —including the point representing a public key, or
   the point forming the first half of a signature (R above)— we reject points
   contained in the small subgroup of the field. A low-order public key breaks
-  unforgeability, and a low-order R leaks private key information. Many
+  non-repudiation, and a low-order R leaks private key information. Many
```

Comment:
Iirc @valerini was exploring the correct terminology and if "non-repudiation" is the idiomatic term here, but iirc I found some references around that: devs and application focused actors, usually prefer the non-repudiation term.
That said I'll approve this PR.